sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/implementation/util/DataLakeImplUtils.java

```diff
@@ -14,4 +17,24 @@ public static String endpointToDesiredEndpoint(String endpoint, String desiredEn
             return endpoint.replaceFirst(currentStringToMatch, desiredStringToMatch);
         }
     }
+
+    public static RuntimeException transformBlobStorageException(RuntimeException ex) {
```

Comment:
This could be removed given that `RuntimeException` is a subclass of `Throwable` so the method below should handle all cases this method would be used.

Reply:
I needed to add this because otherwise I need to cast the Throwable to a RuntimeException. Let me make that method you suggested and see how it affects the need for the two same methods